data/common/vocab_type_map.json

```diff
@@ -0,0 +1,59 @@
+{
+	"Book": "Book",
+	"Broderie": "Embroidery",
+	"Clocks": "Clock",
+	"Clothing": "Clothing",
+	"Decorative Art": "DecArts",
+	"Decorative arts": "DecArts",
+	"Dessin": "Drawing",
+	"dessin": "Drawing",
+	"drawing": "Drawing",
+	"Drawing": "Drawing",
+	"Drawing": "Drawing",
+	"Email": "Enamel",
+	"Émail": "Enamel",
+	"Émail": "Enamel",
+	"Embroidery": "Embroidery",
+	"Enamel": "Enamel",
+	"Furniture": "Furniture",
+	"furniture": "Furniture",
+	"Gemälde": "Painting",
+	"gemalde": "Painting",
+	"Gemalde": "Painting",
+	"GEMÄLDE": "Painting",
+	"GemÃlde": "Painting",
+	"Maps": "Map",
+	"Meuble": "Furniture",
+	"miniatur": "Miniature",
+	"Miniatur": "Miniature",
+	"MINIATUR": "Miniature",
+	"Miniature": "Miniature",
+	"Mosaic": "Mosaic",
+	"Mosaique": "Mosaic",
+	"Mosaïque": "Mosaic",
+	"painting": "Painting",
+	"Painting": "Painting",
+	"peinture": "Painting",
+	"Peinture": "Painting",
+	"Photograph": "Photograph",
+	"Playing Cards": "Cards",
+	"Print": "Print",
+	"sculpture": "Sculpture",
+	"Sculpture": "Sculpture",
+	"skulptur": "Sculpture",
+	"Skulptur": "Sculpture",
+	"tapestry": "Tapestry",
+	"Tapestry": "Tapestry",
+	"tapisserie": "Tapestry",
+	"Tapisserie": "Tapestry",
+	"ZEICHNUNG": "Drawing",
+	"zeichnung": "Drawing",
+	"Zeichnung": "Drawing",
+
+	"Dentelle": "XXX",
```

Comment:
@azaroth42 Should we create new types for these, or just ignore them for now?

Reply:
We might also want to think about a way to gracefully handle case insensitivity in these lookup tables.

Reply:
I'm okay (at least for now) with case sensitive, and the obvious repetition needed.  Such a mapping probably should throw errors when it encounters something with a different case than expected.

For the XXX, I have been putting in HumanMadeObject (or whatever the raw model.Foo class is) so that the pipeline runs through

Reply:
OK. In this case, the code falls back to HMO (for any case where the mapping does not exist). Do you prefer that I include these values in the mapping table where we know it's in the data, but don't have a specific crom type to assert?

Reply:
Since these are mapping into the `vocal` module, and HMO is a `model`-level class, I'm going to just remove these entries, and let the code fall back to HMO.

Reply:
:+1: